src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const listItems = document.querySelectorAll('li');
```

Comment:
It's a good practice to check if listItems are not null before using it in forEach loop.